sdk/core/core/inc/az_span.h

```diff
@@ -153,7 +144,7 @@ AZ_NODISCARD AZ_INLINE az_span _az_span_init(uint8_t* ptr, int32_t size)
  */
 // Force a division by 0 that gets detected by compilers for anything that isn't a byte array.
 #define AZ_SPAN_FROM_BUFFER(BYTE_BUFFER) \
-  _az_span_init( \
+  az_span_init( \
```

Comment:
How would this work when calling the non-inlined az_span_init? Does this result in a `.h -> .c` cycle?

Reply:
Yes - macro gets substituted by a function call.